### DISCUSSION TITLE
Water Plane

### DISCUSSION THREAD

**Discussion by richardanaya** [pmndrs/drei#1278] [Ideas]
### Describe the feature you'd like:

I'm interested in building virtual worlds with Drei, you all seem to have a number of environmental components, I'd like to suggest a water component. 

### Suggested implementation:

Just looking for a large plane that has realistic water effects.  Thank you!

* colors?
* transparency?
* turbulence?

**Comment by drcmda** [comment]
drei is more like a collection of things people once struggled with and made it open source. if you ever figure out how to make that water plane, make sure to make it a pr. :)
